targets/mbed/Makefile.mbed

```diff
@@ -0,0 +1,101 @@
+# Copyright 2015-2016 Samsung Electronics Co., Ltd.
+# Copyright 2016 University of Szeged.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# use TAB-8
+
+CURDIR          = `pwd`
+TYPE            = release
+TARGET_LIST     = k64f stm32f4 stm32f429i nucleo
+JERRYHEAP      ?= 16
+
+ifneq ($(filter $(board), $(TARGET_LIST)),)
+	TARGET        = $(board)
+ifeq ($(TARGET), k64f)
+	YOTTA_TARGET  = frdm-k64f-gcc
+	TARGET_DIR   ?= /media/$(USER)/MBED
+else ifeq ($(TARGET), stm32f4)
+	YOTTA_TARGET  = stm32f4-disco-gcc
+else ifeq ($(TARGET), stm32f429i)
+	YOTTA_TARGET  = stm32f429i-disco-gcc
+else ifeq ($(TARGET), nucleo)
+	YOTTA_TARGET  = st-nucleo-f401re-gcc
+	TARGET_DIR   ?= /media/$(USER)/NODE_F401RE
+endif
+
+UPPERC_TARGET = $(shell echo $(TARGET) | tr a-z A-Z) 
+INTERM        = build/obj-mbed-$(TARGET)
+OUTPUT        = build/bin/$(TYPE).mbed$(TARGET)
+COPYTARGET    = targets/mbed/libjerry
+
+else
+  $(error This board ($(board)) is not supported!)
+endif
+
+EXT_CFLAGS := -D__TARGET_MBED_$(UPPERC_TARGET)
+EXT_CFLAGS += -mlittle-endian -mthumb -mcpu=cortex-m4
+EXT_CFLAGS += -Wno-error=format=
+
+EXT_PORT_DIR := "nasty-hack-nonexistent-directory-to-prevent-jerry-core-from-compiling-any-port-implementation"
```

Comment:
Yes, that's me to blame. I've suggested this in an offline discussion. :)
Then, I've had a probably nicer idea to remove the need for the hack. I see that right now this patch touches the targets directory only. But if you'd add just two lines to jerry-core/CMakeLists.txt to enhance the port_dir processing a bit, then things could become cleaner.:
```
 if(NOT ("${PORT_DIR}" STREQUAL ""))
  file(GLOB SOURCE_PORT_FILES ${PORT_DIR}/*.c)
 endif()
```
Then, `EXT_PORT_DIR:=""` should work.
(I haven't tried it out myself, so this is just a best guess. Let me know whether it works.)